PusherClient/EventEmitter.cs

```diff
@@ -16,7 +16,9 @@ public void Bind(string eventName, Action<dynamic> listener)
         {
             if(_eventListeners.ContainsKey(eventName))
             {
-                _eventListeners[eventName].Add(listener);
+                // To avoid triggering the same bind multiple times

```

Comment:
This comment helps explain what this does. But I'm not sure of the why. Why was this required? Out of interest, more than anything else.